### PR TITLE
Css two column layout | Nick Hernandez

### PR DESCRIPTION
**Pull request by Nickolas-Hernandez**
Screen Shot 2021-01-22 at 3 26 56 PM

### CODE REVIEW

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+/* Layout constructor classes */
+.container {
+  max-width: 1050px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+
+  /* padding: 2rem 0; */
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:14]: Remove commented code before submitting

File: css-two-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  color: #d14c45;
+  border: 2px solid #d14c45;
+}
+
+/* Extra Dipping Sauce */
+button:hover {
+  color: white;
+  background-color: #d14c45;
+  cursor: pointer;
+}
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:74]: nice

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice work so far!! A couple minor changes:

File: css-two-column-layout/styles.css (hunk 1 above)
Comment (Nickolas-Hernandez) [reply css-two-column-layout/styles.css:14]: would this also include an "scaffolding comments" such as noting where global styles are or where certain section styles are?

Comment (Nickolas-Hernandez) [reply css-two-column-layout/styles.css]: Screen Shot 2021-01-22 at 8 07 54 PM

removed commented code

**Review by codingcodymiller — APPROVED** [review]
Nice work!! And to answer your question, comments to label sections are okay, just don't go too far overboard with it. In general, your code should describe itself well enough that comments aren't necessary.

**Pull request merged**